arangod/Replication/TailingSyncer.cpp

```diff
@@ -957,11 +958,22 @@ Result TailingSyncer::changeView(VPackSlice const& slice) {
     }
   }
 
+  // nowadays, the view properties are contained directly inside the
+  // "data" attribute. it is unclear if this was ever different, but
+  // we used to check inside the "properties" attribute as well. so
+  // let's check both
   VPackSlice properties = data.get("properties");
+  if (!properties.isObject()) {
+    properties = data;
+  }
```

Comment:
That's not needed, using just `data` is sufficient. The code with `properties` has never been released.

Reply:
Ok, adjusted.